.github/workflows/ci.yml

```diff
@@ -15,61 +15,70 @@ jobs:
   checks:
     name: Checks
     runs-on: ubuntu-latest
-    steps:
-
-    - name: Setting conda path
-      run: echo "${HOME}/miniconda3/bin" >> $GITHUB_PATH
+    defaults:
+      run:
+        shell: bash -l {0}
 
+    steps:
     - name: Checkout
       uses: actions/checkout@v1
 
     - name: Looking for unwanted patterns
       run: ci/code_checks.sh patterns
       if: always()
 
-    - name: Setup environment and build pandas
-      run: ci/setup_env.sh
-      if: always()
+    - name: Cache conda
+      uses: actions/cache@v1
+      env:
+        CACHE_NUMBER: 0
```

Comment:
what does this do?

Reply:
just fix the hash key?

Reply:
I copied from conda incubator. But I guess we don't need this.